Console/GenerateEncryptionKey.php

```diff
@@ -68,10 +76,17 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        $newKey = null;
         if (!$input->getOption(self::INPUT_KEY_FORCE)) {
             $output->writeln('<info>Run with --force to generate a new key. This will decrypt and reencrypt values in core_config_data and saved credit card info</info>');
             return Cli::RETURN_FAILURE;
         }
+        if ($input->hasOption(self::INPUT_KEY_KEY)) {
+            $newKey = $input->getOption(self::INPUT_KEY_KEY);
+            $output->writeln('<info>The provided crypt key will be used for reencryption.</info>');
+        }else{
```

Comment:
I really need to get a phpcs plugged into this, but if this could be tweaked please

```suggestion
        } else {
```